public/server/db/pr.json

```
@@ -0,0 +1 @@
+{ "definition": "Pull Request"}
```

This looks good, thank you! I'd suggest you follow indentation norms, so that everything looks sort of uniform.
```suggestion
{ 
     "definition": "Pull Request"
 }
```